pkg/visor/init.go

```diff
@@ -186,10 +187,39 @@ func initSNet(ctx context.Context, v *Visor, log *logging.Logger) error {
 		STCP: v.conf.STCP,
 	}
 
+	var nat stun.NATType
+	var host *stun.Host
+	var err error
+stunLoop:
```

Comment:
I guess with the current list of Stun servers, the best thing to do is iterate over them and used the named loop like you do. But I think it should be fine for now to just have one server down the road for this so we can also get rid of this.